editor/src/components/inspector/sections/layout-section/self-layout-subsection/simplified-layout-subsection.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { FlexColumn, FlexRow, InspectorSubsectionHeader } from '../../../../../uuiui'
+import { EditorContractDropdown } from '../../../editor-contract-section'
+import { FixedHugDropdown } from '../../../fill-hug-fixed-control'
+import { UIGridRow } from '../../../widgets/ui-grid-row'
+import { ClipContentControl } from './clip-content-control'
+import { FrameUpdatingLayoutSection } from './frame-updating-layout-section'
+
+export const SimplifiedLayoutSubsection = React.memo(() => {
+  return (
+    <FlexColumn style={{ paddingBottom: 8 }}>
+      <InspectorSubsectionHeader>
+        <FlexRow
+          style={{
+            flexGrow: 1,
+            gap: 8,
+            height: 42,
+          }}
+        >
+          <EditorContractDropdown />
+        </FlexRow>
+      </InspectorSubsectionHeader>
+      <FlexColumn style={{ gap: 8, paddingLeft: 8, paddingRight: 8 }}>
+        <FrameUpdatingLayoutSection />
+        <UIGridRow
+          padded={false}
+          variant='<--1fr--><--1fr-->'
+          style={{ minHeight: undefined, gap: 4 }}
+        >
+          <FixedHugDropdown dimension='width' />
```

Comment:
noice! I think having an inner label here would be clearer: `H`orizontal and `V`ertical would be nice, but there would be a stuttering `H`. What about the dimension control icons?

Reply:
Yeah, this definitely needs either the icons or a label to clarify what these dropdowns apply to